spec/release_spec.rb

```diff
@@ -1,23 +1,18 @@
 # frozen_string_literal: true
 
+require_relative "spec_helper"
 require "fauxpaas/release"
 require "fauxpaas/shell/basic"
 require "fauxpaas/sites"
-require_relative "support/fake_remote_runner"
 
 module Fauxpaas
-  # class FakeWorkingDir
-  #   def initialize(dir, files)
-  #     @dir = dir
-  #     @files = files
-  #   end
-  #   attr_reader :dir
-  #   def relative_files
-  #     @files
-  #   end
-  # end
-
   RSpec.describe Release do
+    before(:each) do
+      Fauxpaas.config.tap do |canister|
+        canister.register(:release_time_format) { Fauxpaas.settings.release_time_format }
+      end
+    end
+
```

Comment:
This seems a little odd to have to do. Are we varying the settings or disposing of the registry somehow I'm not seeing?

Reply:
The registry is filled out in `lib/fauxpaas.rb`, which of necessity requires most of the project. We don't require that as part of the specs specifically to avoid having to bring other classes in. So the registry is empty aside from what we set here.

Reply:
I do agree that there's no real reason not to immediately import the literals from ettin, which would allow us to remove this `before(:each)`.